### PR TITLE
tests/ci: Wipe the workspace when finish the build. to fix the permission denied issue

### PR DESCRIPTION
**Pull request by cpanato**
Try to fixes the permission denied in Jenkins wiping the workspace after finish the job

 * add terminal color
 * Clean the work space after finish each step

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -1,3 +1,5 @@
+#!groovy​
```
Comment (mxinden) [inline Jenkinsfile]: Is this a valid shebang?
http://docs.groovy-lang.org/latest/html/documentation/#_shebang_line

File: Jenkinsfile [hunk 2]
```diff
@@ -519,14 +562,17 @@ pipeline {
...
+                  ansiColor('xterm') {
+                    checkout scm
+                    unstash 'installer'
+                    unstash 'node_modules'
+                    sh """#!/bin/bash -ex
+                    cd installer
+                    make launch-aws-installer-guitests
+                    make gui-aws-tests-cleanup
+                    """
+                    deleteDir()
```
Comment (mxinden) [inline Jenkinsfile]: Not sure if this runs even though previous lines like `sh` failed. Do you know @cpanato?

File: Jenkinsfile (hunk 1 above)
Comment (cpanato) [inline Jenkinsfile]: it works on my ide :) but I can change if you want

File: Jenkinsfile (hunk 2 above)
Comment (cpanato) [inline Jenkinsfile]: yes, it runs

File: Jenkinsfile (hunk 1 above)
Comment (mxinden) [inline Jenkinsfile]: Ok, just leave it like this. I don't have a strong opinion about it.

**Comment by mxinden** [comment]
@cpanato Can you resolve the merge conflicts?

Once that is done: @Quentin-M @squat or @estroz feel free to merge if you have no objections.

**Comment by cpanato** [comment]
@mxinden done

**Comment by mxinden** [comment]
@squat If you have no concerns here, feel free to merge. Thanks for all the reviews!

**Comment by squat** [comment]
Looks great now!

**Comment by mxinden** [comment]
^ @squat thanks.

**Pull request merged**